README.md

```diff
@@ -4,4 +4,5 @@ Contributors
 
 1. Aditya Agarwal
 2. XYZ
+2. Rahil
```

Comment:
Do these things->

1. Get the latest code of my master branch to your `patch-1` branch. The command for that would be

```sh
git checkout patch-1 # switch to yash branch
git pull upstream master # get the code of upstream's master branch in your local's patch-1 branch.
```

If the second command gives an error then most likely you have not added the upstream remote to your local git repo. To do that use this command

```sh
git remote add upstream https://github.com/itaditya/road-to-hacktober.git
```

After you do this you'll get a merge conflict. Why did that happen? Because Khusharth's name was added and now GitHub doesn't know how to combine your code and Khusharth code that has been merged. So it will ask you to fix it.

2. After merging conflicts, the final code of Readme.md should be

```md
## Repo to let people try making Pull Requests

### Contributors

1. Aditya Agarwal
1. Khusharth Patani
1. Rahil
```

If Yash's code gets merged before yours then you have to again pull the latest code from upstream master then fix merge conflicts and then commit that code.